### ISSUE TITLE
Browser tab freeze if element with v-maska is hidden with v-if

### ISSUE THREAD

**Issue by anatolykopyl** [beholdr/maska#77]
Can be avoided by using `v-show` instead, but this is not a perfect solution.

For example I want to have a search input that is masked when I'm searching for hashtags and an unmasked input for everything else:
```vue
<template>
  <input
    v-if="category === 'hashtags'"
    v-model="query"
    v-maska="hashtagMask"
    type="text"
    placeholder="search"
  >
  <input
    v-else
    v-model="query"
    type="text"
    placeholder="search"
  >
</template>
```

```vue
<script>
export default {
  data() {
    return {
      category: 'all',
      query: '',
      hashtagMask: { mask: '!#H*', tokens: { H: { pattern: /[0-9a-zA-Zа-яА-Я_]/ } } }
    }
  }
}
</script>
```

This component will freeze the browser upon switching categories and entering text into the input without the `v-maska` directive.

**Comment by krinteron** [comment]
ran into this problem on the Vue 3 version too

**Comment by krinteron** [comment]
> 

ran into this problem on the Vue 3 version too

crutch:
````
updated() {
    this.$refs.input.setAttribute('data-mask-raw-value', true);
    this.$refs.input.setAttribute('data-mask-inited', true);
    this.$refs.input.setAttribute('data-mask', this.activeMask);
  }
````

**Comment by beholdr** [comment]
I have released new major version (v2.0). If this issue still valid, please reopen it.
